bundles/org.openhab.binding.shelly/README.md

```diff
@@ -823,7 +803,128 @@ Number Shelly_Power     "Bath Room Light Power"                {channel="shelly:
 
 ### shelly.rules
 
-reading colors from color picker:
+#### Catch alarms
+
+```
+rule "Monitor Shelly Restartt"
+when
+    Channel "shelly:shelly2-relay:XXXXXX:device#alarm" triggered OVERTEMP
+then
+        logInfo("Shelly1", "Device is getting to hot!!")
+end
+```
+
+#### Trigger scene with Button-1
+
+```
+rule "Button-1 SHORT_PRESSED"
+when
+    Channel "shelly:shellybutton1:d8f15bXXXXXX:status#button" triggered SHORT_PRESSED
+then
+    logInfo("Button", "Shelly Button reported SHORT_PRESSED")
+    if (MyTV.state != OFF) {
+        logInfo("Button", "   switch TV OFF")
+        sendCommand(MyTV, "OFF")
+    } else {
+        logInfo("Button", "   switch TV to ON")
+        sendCommand(MyTV, ON)
+    }
+end
+
+rule "Button-1 TRIPLE_PRESSED"
+when
+    Channel "shelly:shellybutton1:d8f15bXXXXXX:status#button" triggered TRIPLE_PRESSED
+then
+    logInfo("Button", "Shelly Button reported TRIPLE_PRESSED")
+end
+```
+
+#### Observe battery status
+
+pre-requisites:
+
+- Install Send Mail Action
+- Define a group called gBatteries
+Group   gBattery        "Batterien"         <battery>       (All)
```

Comment:
The lower than and greater than signs need to be escaped, or better the entire line:
```suggestion
`Group   gBattery        "Batterien"         <battery>       (All)`
```

Reply:
done